api/IDBVersionChangeEvent.json

```diff
@@ -44,10 +44,10 @@
             "version_added": true
           },
           "safari": {
-            "version_added": "7"
+            "version_added": "6.1"
           },
           "safari_ios": {
-            "version_added": true
+            "version_added": "7"
```

Comment:
Is this "7" is mistake? Are any other IDB* APIs left in iOS < 8 after this change?

Reply:
It is -- correcting now!